ManageWikiSettings.php

```diff
@@ -1338,6 +1338,15 @@
 		'section' => 'styling',
 		'help' => 'If not set, then the img.line element (white divider line in the top menu, between the site name and the down arrow/"Upload file" link) won\'t be generated.',
 	],
+	'wgMinervaEnableSiteNotice ' => [
+		'name' => 'Minerva Sitenotice',
+		'from' => 'mobilefrontend',
+		'restricted' => false,
+		'type' => 'check',
+		'overridedefault' => true,
```

Comment:
Um, that should be in false, right? 'Cause you check the box in ManageWiki.

Reply:
Nope, this makes sure it is true by default, allowing users to unset it.

If you set this as false it'll mean it defaults to false.

Reply:
Okay, I wanted to make sure. Thank you.